Gemfile

```diff
@@ -6,12 +6,6 @@ ruby '3.1.3'
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225282233-7122ad96-91b9-4dc1-8d73-142aace2c436.png)

**[Optional]**

- [ ] It would be nice if you make your message like "If you like this project then don't forget to share it and give star :star: on this repository" to make your readme professional :blush: